testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -488,7 +510,23 @@ def test_bucket_online_offline_resync_with_offline(params_from_base_test_setup,
         log_info("resync issued !!!!!!")
     except Exception as e:
         log_info("Catch resync exception: {}".format(e))
-
+    if sync_gateway_version < "3.0.0":
+        resync_result = async_resync_result.get()
```

Comment:
is this section of code same as the other code blocks above? if so, shall consider to have a common function to handle this piece?

Reply:
yes, we can move it as a helper method.

Reply:
done